stable/quassel/README.md

```diff
@@ -94,7 +94,7 @@ $ helm install stable/quassel --name my-release -f values.yaml
 ## Persistence
 
 The [quassel-core](https://hub.docker.com/r/linuxserver/quassel-core) image
-stores it's configuration data, and if using SQLite, it's SQLite datbase at the
+stores it's configuration data, and if using SQLite, it's SQLite datebase at the
```

Comment:
You have a typo in your typo fix :) It should be `database`, ofcourse.

Reply:
Thanks for your correct!